### ISSUE TITLE
Site root not resolved when moving to different server

### ISSUE THREAD

**Issue by normington** [alexweissman/UserFrosting#261]
I've tried to pick up my UserFrosting installation and move it to a different server, by copying all the files over and using an SQL dump exported from phpMyAdmin to set up the database on the new server.

I remembered to change the settings in `models/db_functions.php` to those of the database on the new server. I also changed the `website_url` value in `uf_configuration`.

I figured this should be all I'd need to do to make it work, but what I'm seeing is that no links work (including pulling in source files), because it seems the site root path is not being resolved properly.

For example, the `<head>` section of the `index.php` gets rendered like this:
```
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="#DESCRIPTION#">
  <meta name="author" content="#AUTHOR#">
  <link rel="shortcut icon" href="css/favicon.ico">

  <title> | Welcome</title>

  <link rel="shortcut icon" type="image/x-icon" href="css/favicon.ico" />
  
  <!-- Bootstrap core CSS -->
  <link href="http://css/bootstrap.css" rel="stylesheet">

  <!-- Custom styles for this template -->
  <link href="http://css/jumbotron-narrow.css" rel="stylesheet">
  
  <link rel="stylesheet" href="http://css/font-awesome.min.css">
  <link rel="stylesheet" href="http://css/bootstrap-switch.min.css" type="text/css" />
   
  <!-- JavaScript -->
  <script src="http://js/jquery-1.10.2.min.js"></script>
  <script src="http://js/bootstrap.js"></script>
  <script src="http://js/userfrosting.js"></script>
  <script src="http://js/date.min.js"></script>
  <script src="http://js/handlebars-v1.2.0.js"></script> 

</head>
```

Clearly, something's going wrong with the preprocessing, but I'm not sure what. Would you be able to give me some pointers in troubleshooting?

* The old server was XAMPP running locally, with PHP 5.5. The new server is shared hosting, with PHP 5.4.
* I wanted to check that it wasn't just something wrong with the server config, so I tried doing a new installation of UserFrosting (from the same zip downloaded from GitHub that I used to install my original one) into a different database but all running on the same server.
  * I did get a warning during the installation that PHP 5.5 was recommended, but that PHP 5.4 would be okay
  * The installer completed successfully and the clean installation is working fine.
* The version I'm using of UserFrosting for the original installation, which I'm trying to copy to the new server, and for the clean install that I did to check if it was a server config issue, was that of d9b8523.

**Comment by alexweissman** [comment]
Hmm, my guess is that it's pulling an empty value from `website_url`.  Can you verify that it's connecting to the DB after install?

**Comment by normington** [comment]
__False alarm, it was just me being silly.__

![I've made a huge mistake!](http://i3.kym-cdn.com/photos/images/masonry/000/639/484/4d3.jpg)

I tried to see if it was connecting to the DB okay by dropping the install directory back in, and the install script made it clear to me that it couldn't connect to the database at all.

It turns out I was doing too much multitasking when I did the DB setup, and forgot to give the user any permissions. Fixed now and it all works a dream - sorry for the confusion, I'll have more a think next time!
